pkg/trie/triedb/triedb_iterator.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+import (
+	"bytes"
+
+	nibbles "github.com/ChainSafe/gossamer/pkg/trie/codec"
+	"github.com/ChainSafe/gossamer/pkg/trie/triedb/codec"
+)
+
+type iteratorState struct {
+	parentFullKey []byte     // key of the parent node of the actual node
+	node          codec.Node // actual node
+}
+
+// fullKeyNibbles return the full key of the node contained in this state
+// child is the child where the node is stored in the parent node
+func (s *iteratorState) fullKeyNibbles(child *int) []byte {
```

Comment:
Add a full stop to differentiate both comments; when we hover over the function, it shows continuous text.
![Screenshot from 2024-04-26 12-47-06](https://github.com/ChainSafe/gossamer/assets/40173579/e8407050-47ae-4a1d-803c-6936f6015a06)
```suggestion
// fullKeyNibbles return the full key of the node contained in this state.
// child is the child where the node is stored in the parent node
func (s *iteratorState) fullKeyNibbles(child *int) []byte {
```